rfc/accepted/0001-new-rfc-process/index.md

```diff
@@ -0,0 +1,22 @@
+<!--
+Copyright (c) The OpenTofu Authors
+SPDX-License-Identifier: MPL-2.0
+-->
+
+# RFC-0001: New RFC process
+
+## Abstract
+
+This RFC adds a new RFC process that enables better conversations and longer term collaboration between the core team and the community.
+
+## Use case
+
+The new RFC process is pull request-based, which means that participants can start conversations on individual lines of text rather than just having one thread in GitHub issues. It also enables core engineers to add changes to the text by committing into the source branch of the RFC, as well as by submitting PRs against the source branch.
```

Comment:
Potential layout for this section:

* Current process and discuss where it falls short
* Ad-hoc evolution for state encryption
* Refining and formalizing the process